tests/data/dummy_protocol/protocol.yaml

```diff
@@ -6,10 +6,7 @@ type: protocol
 description: The scaffold protocol scaffolds a protocol to be implemented by the developer.
 license: Apache-2.0
 aea_version: '>=1.0.0, <2.0.0'
-fingerprint:
-  __init__.py: Qmc9Ln8THrWmwou4nr3Acag7vcZ1fv8v5oRSkCWtv1aH6t
-  message.py: QmWPrVTSHeKANCaVA4VaQyMGLix7yiMALbytsKZppAG2VU
-  serialization.py: QmaAf5fppirUWe8JaeBbsqfbeofTHe8DDGHJooe2X389qo
+fingerprint: {}
```

Comment:
What has happened here?

Reply:
https://github.com/valory-xyz/open-aea/tree/b014d5ac7ea4382cf85311754623830121b97712/tests/data/dummy_protocol

There are no files except the `protocol.yaml` in `tests/data/dummy_protocol` that's why there are no fingerprints in this config file